add contributors for ambient (#11841)
istio/istio.io: content/en/blog/2022/introducing-ambient-mesh/index.md

A build of Istio which supports ambient mesh is available to [download and try](/blog/2022/get-started-ambient/) in the [Istio Experimental repo]({{< github_raw >}}/tree/experimental-ambient).
A list of missing features and work items is available in the [README]({{< github_raw >}}/blob/experimental-ambient/README.md).
Please try it out and let us know what you think!

> *Contributors: Aaron Birkland (Solo.io), Craig Box (Google), Greg Hanson (Solo.io), Daniel Hawton (Solo.io), John Howard (Google), Ethan J. Jackson (Google), Denis Jannot (Solo.io), Abhi Joglekar (Google), Yuval Kohavi (Solo.io), Steven Landow (Google), Idit Levine (Solo.io), Oliver Liu (Google), Yossi Mesika (Solo.io), Justin Pettit (Google), Neeraj Poddar (Solo.io), Nina Polshakova (Solo.io), Christian Posta (Solo.io), Doug Reid (Google), Louis Ryan (Google), Lin Sun (Solo.io), Eitan Yarmush (Solo.io), Kuat Yessenov (Google), Francis Zhou (Google)*

Reviewer: ```suggestion
*Thank you to the team that contributed to the launch of ambient mesh: Aaron Birkland, Craig Box, Greg Hanson, Daniel Hawton, John Howard, Ethan J. Jackson, Denis Jannot, Abhi Joglekar, Yuval Kohavi, Steven Landow, Idit Levine, Oliver Liu, Yossi Mesika, Justin Pettit, Neeraj Poddar, Nina Polshakova, Christian Posta, Doug Reid, Louis Ryan, Lin Sun, Eitan Yarmush, Kuat Yessenov and Francis Zhou*
```

Reviewer: That list is an eyesore with the affiliations.  With so many names, it's also better for the community to adjust it in the proposed fashion.

Author: I can take your other suggestions but i think we have agreed to put company names.

> *Thank you to the team that contributed to the launch of ambient mesh: Aaron Birkland (Solo.io), Craig Box (Google), Greg Hanson (Solo.io), Daniel Hawton (Solo.io), John Howard (Google), Ethan J. Jackson (Google), Denis Jannot (Solo.io), Abhi Joglekar (Google), Yuval Kohavi (Solo.io), Steven Landow (Google), Idit Levine (Solo.io), Oliver Liu (Google), Yossi Mesika (Solo.io), Justin Pettit (Google), Neeraj Poddar (Solo.io), Nina Polshakova (Solo.io), Christian Posta (Solo.io), Doug Reid (Google), Louis Ryan (Google), Lin Sun (Solo.io), Eitan Yarmush (Solo.io), Kuat Yessenov (Google) and Francis Zhou (Google)*

Reviewer: ```suggestion
*Thank you to the team that contributed to the launch of ambient mesh! 
* Google: Craig Box, John Howard, Ethan J. Jackson, Abhi Joglekar, Steven Landow, Oliver Liu, Justin Pettit, Doug Reid, Louis Ryan, Kuat Yessenov, Francis Zhou
* Solo.io: Aaron Birkland, Greg Hanson, Daniel Hawton, Denis Jannot, Yuval Kohavi, Idit Levine, Yossi Mesika, Neeraj Poddar, Nina Polshakova, Christian Posta, Lin Sun, Eitan Yarmush
```

Author: @justinpettit was very much against of listing under two companies which was the initial idea.

Reviewer 2: I was, but there are now a lot more contributors listed than I thought we'd have.  If we're not going to trim the list, I'd prefer Craig's suggestion.

Author: ok cool! that is fine with me too!  Thanks to both!!

> *Thank you to the team that contributed to the launch of ambient mesh: Aaron Birkland (Solo.io), Craig Box (Google), Greg Hanson (Solo.io), Daniel Hawton (Solo.io), John Howard (Google), Ethan J. Jackson (Google), Denis Jannot (Solo.io), Abhi Joglekar (Google), Yuval Kohavi (Solo.io), Steven Landow (Google), Idit Levine (Solo.io), Oliver Liu (Google), Yossi Mesika (Solo.io), Justin Pettit (Google), Neeraj Poddar (Solo.io), Nina Polshakova (Solo.io), Christian Posta (Solo.io), Doug Reid (Google), Louis Ryan (Google), Lin Sun (Solo.io), Eitan Yarmush (Solo.io), Kuat Yessenov (Google) and Francis Zhou (Google)*

Reviewer 3: As an FYI, I broke this lint into multiple lines to get past the lint failure and it still showed as italics:
```suggestion
*Thank you to the team that contributed to the launch of ambient mesh: Aaron Birkland (Solo.io), Craig Box (Google),
Greg Hanson (Solo.io), Daniel Hawton (Solo.io), John Howard (Google), Ethan J. Jackson (Google), Denis Jannot (Solo.io),
Abhi Joglekar (Google), Yuval Kohavi (Solo.io), Steven Landow (Google), Idit Levine (Solo.io), Oliver Liu (Google),
Yossi Mesika (Solo.io), Justin Pettit (Google), Neeraj Poddar (Solo.io), Nina Polshakova (Solo.io),
Christian Posta (Solo.io), Doug Reid (Google), Louis Ryan (Google), Lin Sun (Solo.io), Eitan Yarmush (Solo.io),
Kuat Yessenov (Google) and Francis Zhou (Google)*
```

Author: Yes, that is great to know! Good way to get the lint fixed.
